splunk/scripts/index_verification.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+logs_path="$SPLUNK_HOME/test_logs"
+endpoint="https://localhost:8089/services/data/inputs/oneshot"
+
+echo "Indexing logs..."
+for file in "$logs_path"/*; do
+  echo "Indexing $file"
+  curl -k -u admin:"$SPLUNK_PASSWORD" "$endpoint" -d name="$file"
+done
+
+response=$(curl -k -u admin:"$SPLUNK_PASSWORD" "$endpoint" -X GET -d output_mode=json)
```

Comment:
Can use `--get` as a higher-level alternative to `-X GET`

Reply:
done